tests/0011-produce_batch.c

```diff
@@ -90,8 +90,8 @@ static void test_single_partition(void) {
         int failcnt = 0;
         int i;
         rd_kafka_message_t *rkmessages;
-        const int topic_suffix_length = 56, client_id_length = 239;
-        char *topic_suffix, *client_id;
+        const int client_id_length = 239;
+        char *client_id;
```

Comment:
Can simplify it with a memset, see below. In C I only saw it happening from 270 on.
```c
        char client_id[271];
```